src/org/etools/j1939_84/controllers/part01/Part01Step26Controller.java

```diff
@@ -300,47 +307,618 @@ protected void run() throws Throwable {
+        // @Joe we just need a method to call here. Eric added a column to Table A-1, I believe; please double check me
+        // on that
+        // FIXME: the call to then new Table A-1 validator method for this functionality need to be added here once the
+        // method has been written - note: call may need to move into obdModule loop
+    }
+
+    private void testSp12783(OBDModuleInformation module) {
+        // 6.1.26.23 Actions12 for MY2022+ Plug-in HEV DRIVES
+        // a. DS request message to ECU that indicated support in DM24 for upon request
+        // SP 12783 (Hybrid Lifetime Distance Traveled in Charge Depleting Operation with
+        // Engine off) for
+        // PG 64244 Hybrid Charge Depleting or Increasing Operation Lifetime Hours
+        int[] ghgLifeTimeSps = { 64244 };
+        List<GenericPacket> ghgLifeTimePackets = requestPackets(module.getSourceAddress(),
+                                                                ghgLifeTimeSps)
+                                                                               .stream()
+                                                                               // 6.1.26.23.b.
+                                                                               // Record
+                                                                               // each
+                                                                               // value
+                                                                               // for
+                                                                               // use
+                                                                               // in Part
+                                                                               // 12.
+                                                                               .peek(this::save)
+                                                                               .collect(Collectors.toList());
+
+        for (int pg : ghgLifeTimeSps) {
+            GenericPacket packetForPg = haveResponseWithPg(ghgLifeTimePackets, pg);
+            if (packetForPg == null) {
+                // 6.1.26.24.a. Fail PG query where no response was received
+                addFailure("6.1.26.24.a - No response was received from "
+                        + module.getModuleName() + "for PG "
+                        + pg);
+            } else {
+                packetForPg.getSpns()
+                           .forEach(spn -> {
+                               // SAE INTERNATIONAL J1939TM-84 Proposed Draft 24 March
+                               // 2022 Page 37 of 140
+                               // 6.1.26.24.b - Fail PG query where any accumulator value
+                               // received is greater than FAFFFFFFh.
+                               if (spn.getRawValue() >= 0xFAFFFFFFL) {
```

Comment:
I am confused because according to the "da-extract.csv", PGs 64244, 64245 & 64246 are active, stored and lifetime values.  I do not see how to filter index values out.